src/Maestro/maestro-angular/src/app/page/build-graph-table/build-graph-table.component.ts

```diff
@@ -96,11 +99,47 @@ function sortBuilds(graph: BuildGraph): BuildData[] {
   result = result.reverse();
   for (const node of result) {
     node.isToolset = isToolset(node.build, graph);
+    node.hasIncoherentDependencies = hasIncoherentDependencies(node.build, result, false);
+    node.hasIncoherentDependenciesIncludingToolsets = hasIncoherentDependencies(node.build, result, true);
   }
 
   return result;
 }
 
+function hasIncoherentDependencies(build: Build, buildData: BuildData[], includeToolsets: boolean): boolean {
```

Comment:
Is this something we should pre caclulate in the "sortBuilds" method? It seems like a lot of work to do for every row on every render of the page.

Reply:
This calculation occurs in the sortBuilds method :)

Reply:
OH! I was confused because of the other method with the same name. :-)